src/networkProvider/constants.ts

```diff
@@ -0,0 +1,3 @@
+import BigNumber from "bignumber.js";
+
+export const MaxUint64 = new BigNumber("18446744073709551615");
```

Comment:
can you rename it to `MaxUint64AsBigNumber` ?

Reply:
Fixed.